source/brailleTables.py

```diff
@@ -788,6 +885,10 @@ def initialize():
 					"Error while applying custom braille tables config from scratchpad manifest: "
 					f"{manifestPath}",
 				)
+	if config.conf["braille"]["translationTable"] == "auto":
+		config.conf["braille"]["translationTable"] = getDefaultTableForCurLang(TableType.TRANSLATION)
```

Comment:
Now, how is the auto support supposed to behave? I see two options:

1. The option you implemented currently, the config spec provides `auto` as default, but as soon as NVDA started for the first time, it decides for an automatic table and then saves that to config. Basically, this means that for a fresh installation of NVDA, the default table will always be the table that corresponds with the language of Windows, which is not necessarily the language the user prefers. If the user then decides to change the language of NVDA, the tables yet stay the same as they were before.
2. Make automatic an option available from the GUI. This means that the automatic table is always based on NVDA's current language and will be reconsidered on a change of the language.